pymetis/__init__.py

```diff
@@ -96,7 +98,7 @@ def nested_dissection(adjacency=None, xadj=None, adjncy=None):
 
 
 def part_graph(nparts, adjacency=None, xadj=None, adjncy=None,
-        vweights=None, eweights=None, recursive=None, contiguous=None):
```

Comment:
Sorry, I should have made myself clear. This set of changes cannot break compatibility with existing code. I did that in #28 to demonstrate, but it cannot be merged while still breaking compat.

Reply:
Sure, that makes sense. It's a quick fix.